docs/source/user-guide/mission-modeler-guide/configuration/index.rst

```diff
@@ -2,9 +2,14 @@
 Configuring a Mission Model
 ===========================
 
+A **mission model configuration** enables mission modelers to set initial mission model values when running a simulation.
+Configurations are tied to a plan, therefore each plan is able to define its own set of configuration parameters.
+The ``examples/banananation`` project contains a ``Configuration`` data class example to demonstrate how a simple configuration may be created.
```

Comment:
Minor suggestion, feel free to ignore: can we link to [`examples/bananation`](https://github.com/NASA-AMMOS/aerie/tree/develop/examples/banananation) here? It would entail changing \`\`examples/bananation\`\` to |examples/bananation|_, and then adding to the bottom of the file these two lines:

```
.. |examples/bananation| replace:: ``examples/bananation``
.. _examples/bananation: https://github.com/NASA-AMMOS/aerie/tree/develop/examples/banananation
```